views/js/runner/plugins/content/dialog/itemInlineMessage.js

```diff
@@ -0,0 +1,138 @@
+/**
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2016  (original work) Open Assessment Technologies SA;
+ *
+ * @author Alexander Zagovorichev <zagovorichev@1pt.com>
+ */
+
+/**
+ * Create form with message in test runner and replace all control buttons on his personal
+ */
+
+define([
+    'jquery',
+    'i18n',
+    'ui/hider',
+    'taoTests/runner/plugin',
+    'tpl!taoQtiTest/runner/plugins/navigation/button'
+], function($, __, hider, pluginFactory, buttonTpl) {
+    'use strict';
+
+    /**
+     * The display of the next button
+     */
+    var buttonData = {
+        next : {
+            control : 'move-forward',
+            title   : __('Submit and go to the next item'),
+            icon    : 'forward',
+            text    : __('OK')
+        },
+        end : {
+            control : 'move-end',
+            title   : __('Submit and go to the end of the test'),
+            icon    : 'fast-forward',
+            text    : __('OK & End test')
+        }
+    };
+
+    /**
+     * Create the button based on the current context
+     * @param {Object} selfPlugin
+     * @param {Object} context - the test context
+     * @returns {*|jQuery|HTMLElement} the button
+     */
+    var createOkButton = function createElement(selfPlugin, context){
```

Comment:
Move this function into `init`so you can benefit of the closure's context instead of giving the context as 1st parameter